mslib/msui/mscolab.py

```diff
@@ -530,8 +552,10 @@ def handle_work_locally_toggle(self):
             if self.version_window is not None:
                 self.version_window.close()
             self.create_local_project_file()
-            self.local_ftml_file = fs.path.join(self.data_dir, 'local_mscolab_data',
```

Comment:
I do not understand, from the documentation of fs, why this change is necessary. It seems to be the same, but more complicated...?